config/locales/default.en.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
This file feels much better. A file for default/shared/global/generic pieces of content that do not require context of any particular model or view.